conn.go

```diff
@@ -161,6 +170,8 @@ func newConn(tr transport) (*Conn, error) {
 	conn.nextSerial = 1
 	conn.serialUsed = map[uint32]bool{0: true}
 	conn.busObj = conn.Object("org.freedesktop.DBus", "/org/freedesktop/DBus")
+	conn.shutdownSigWorker = make(chan int, 1)
```

Comment:
When the value doesn't matter, we use an empty struct rather than an int or bool.